src/lib.rs

```diff
@@ -97,59 +97,64 @@ impl From<Error> for io::Error {
     }
 }
 
-// File: 内建文件类型
 // size: 文件大小
-// offset: 文件偏移（目前指针位置），无需序列化
 // start_at: 文件在 source 中的开始位置
-// mime: 文件类型的 MIMETYPE
+#[derive(Serialize, Deserialize, Debug)]
```

Comment:
What is FileHeader for? It is recommended to indicate the purpose and source in the comments (for example, which function returns)

Reply:
In addition, if you want to express the file size in the structure named header, I don't think you should use "size"